test/e2e/runner/load.go

```diff
@@ -134,10 +134,12 @@ func createTxBatch(ctx context.Context, txCh chan<- types.Tx, testnet *e2e.Testn
 			}
 		}()
 	}
+FOR_LOOP:
 	for i := 0; i < testnet.LoadTxBatchSize; i++ {
 		select {
 		case genCh <- struct{}{}:
 		case <-ctx.Done():
+			break FOR_LOOP
```

Comment:
can we just do `break`?

Reply:
`break` would only break from `select`, not the enclosing `for` loop.